docs/current/sdk/elixir/950800-index.md

```diff
@@ -0,0 +1,78 @@
+---
+slug: /sdk/elixir
+---
+
+# Dagger Elixir SDK
+
+<div class="status-badge">Technical Preview</div>
+
+## What is the Dagger Elixir SDK?
+
+The Dagger Elixir SDK contains everything you need to develop CI/CD pipelines in Elixir, and run them with the [Dagger Engine][dagger].
+
+This SDK contains:
+
+* A Elixir package: `dagger.io/dagger`
```

Comment:
Where is this package hosted? How would a user install it?

Reply:
All good, now captured in the `Install` sub-page, as well as `Step 2` of `Get Started` page.